packages/@o3r/eslint-config-otter/rules/typescript/unicorn.cjs

```diff
@@ -6,6 +6,7 @@ module.exports = {
   ],
   rules: {
     'unicorn/prefer-node-protocol': 'error',
-    'unicorn/switch-case-braces': 'warn'
+    'unicorn/switch-case-braces': 'warn',
```

Comment:
Open discussion: Why do we keep warning and not use error everywhere ?

Reply:
To avoid breaking change: *warn* allowes the `--fix` to process to fix of the code but does not break the eslint validation.